WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormDataModel.swift

```diff
@@ -55,6 +55,12 @@ protocol ProductFormDataModel {
     var soldIndividually: Bool? { get }
     // Whether stock status is available for the product.
     func isStockStatusEnabled() -> Bool
+
+    // Product downloads
+    func isDownloadable() -> Bool
```

Comment:
This can be probably transformed in `var downloadable` since we check only the product downloadable property

Reply:
Yeap, changed